dashboard/libs/figure_gen.py

```diff
@@ -1,4 +1,8 @@
+from datetime import date
+
 from dashboard.libs.queries import get_areas, get_persons, get_all_projects, get_tasks, get_dates
+from dashboard.libs.date_tools import get_workdays_list
+from dashboard.libs.rate_generator import get_reference_rate
```

Comment:
we should only use the `rate_generator` in the command for generating fake rates. once done, we should query the database to get the rate instead of generating them again on the fly. feel free to address this later.